proto/rill/runtime/v1/queries.proto

```diff
@@ -404,6 +404,7 @@ message MetricsViewAggregationMeasure {
     MetricsViewAggregationMeasureComputeComparisonDelta comparison_delta = 8;
     MetricsViewAggregationMeasureComputeComparisonRatio comparison_ratio = 9;
     MetricsViewAggregationMeasureComputePercentOfTotal percent_of_total = 10;
+    MetricsViewAggregationMeasureURI uri = 11;
```

Comment:
The type name should be `MetricsViewAggregationMeasureComputeURI` for consistency with the other "compute" options.